gist/src/main/java/build/gist/presentation/GistView.kt

```diff
@@ -25,15 +25,17 @@ class GistView @JvmOverloads constructor(
     attrs: AttributeSet? = null
 ) : FrameLayout(context, attrs), EngineWebViewListener {
 
-    private var engineWebView: EngineWebView = EngineWebView(context)
+    private var engineWebView: EngineWebView? = EngineWebView(context)
```

Comment:
In this file, I don't see a scenario where `engineWebView` would be null? Curious why this variable was changed to optional?